### ISSUE TITLE
Have video tutorial ?

### ISSUE THREAD

**Issue by QPLAYid** [natsumerinchan/libhoudini-package#10]
Vide tutorial install libhoudini-paket

**Comment by bog4t** [comment]
Yeah, I'm also very confused as to how we go about building the kernel and where we can find device/generic/common.
